src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonManager.cs

```diff
@@ -612,8 +638,7 @@ private void GetNextOldestChange()
         {
             var newOldest = handOverTo?.Path.Address;
             Log.Info("Singleton terminated, hand-over done [{0} -> {1}]", _cluster.SelfAddress, newOldest);
-            handOverTo?.Tell(HandOverDone.Instance);
```

Comment:
Why you have remove it?
https://github.com/akka/akka/blob/v2.5-M2/akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala#L756